### ISSUE TITLE
Chars of text layer has varoius shift in Y axis when font is small 

### ISSUE THREAD

**Issue by marcinw788** [airbnb/lottie-web#1552]
Chars exported as svg. In that case there is only one layer - text layer with 15px font. Tried diffrent styles.

![image](https://user-images.githubusercontent.com/14854458/55082000-2a3b5900-50a1-11e9-8db6-d3674f5ea402.png)

Bodymovin / lottie-web 5.4.4 but problem is still on 5.5.0. HTML renderer. Ae 2018, Windows.

**Comment by bodymovin** [comment]
Hi, can you share the .aep?

**Comment by bodymovin** [comment]
I'll look into it.
Any reason why you use the hybrid renderer instead of the svg?
Also, could you export text as text instead of glyphs for your use case? It should look ok with real text.

**Comment by marcinw788** [comment]
I'm working with motion designers and I found that scaling animations (for text layers) didn't work on svg. Also I think that hybrid renderer is a bit fater and performace is critical for my case. 
Also I tried to export text as text but can't atach webfont. But here also I have no control what font they will use (if it will be webfont or not).

**Comment by marcinw788** [comment]
Did you checked my case? I looked into devtools and I think that it could be something with css-s. I don't know if mine or bodymovin. When I changed display: block to flex or grid on div with svg-dot element inside it jumps to font baseline. Despite this I don't know if it is solution becouse of support by older browsers of flex. Or there is some problem with calculation of tranform: translate(x,y) on svg element.
